AsyncDisplayKit/ASTableView.mm

```diff
@@ -787,11 +787,12 @@ - (CGSize)viewportSizeForRangeController:(ASRangeController *)rangeController
 - (ASInterfaceState)interfaceStateForRangeController:(ASRangeController *)rangeController
 {
   ASTableNode *tableNode = self.tableNode;
-  if (tableNode) {
+  if (tableNode && [tableNode supportsRangeManagedInterfaceState]) {
+    // Only use the interfaceState of nodes that are range managed
     return self.tableNode.interfaceState;
   } else {
-    // Until we can always create an associated ASTableNode without a retain cycle,
-    // we might be on our own to try to guess if we're visible.  The node normally
+    // For not range managed nodes or until we can always create an associated ASTableNode
+    // without a retain cycle, we might be on our own to try to guess if we're visible.  The node normally
     // handles this even if it is the root / directly added to the view hierarchy.
     return (self.window != nil ? ASInterfaceStateVisible : ASInterfaceStateNone);
```

Comment:
Both here and in collcetion, let's make the first case in the ternary be (ASInterfaceStateVisible | ASInterfaceStateDisplay)